server/app/controllers/admin/AdminProgramBlocksController.java

```diff
@@ -113,7 +115,7 @@ public Result edit(Request request, long programId, long blockId) {
     try {
       ProgramDefinition program = programService.getProgramDefinition(programId);
       BlockDefinition block = program.getBlockDefinition(blockId);
-      return renderEditViewWithMessage(request, program, block, "");
+      return renderEditViewWithMessage(request, program, block, Optional.empty());
```

Comment:
suggestion (blocking): please add a param name comment for the empty optional i.e. `/* message= */ Optional.empty()`

Reply:
Done.